### ISSUE TITLE
Adding Bundles to cart does not let me specify product attributes

### ISSUE THREAD

**Issue by MatteoPiovanelli-Laser** [bleroy/Nwazet.Commerce#119]
Looks like it's currently impossible to select attributes for products in a bundle when adding the bundle to the cart.

Am I missing something?

**Comment by bleroy** [comment]
No, that's entirely possible. I don't remember ever having to configure the individual products in a bundle. Attributes on the bundle itself, yes, but not on the products in the bundle.

**Comment by bleroy** [comment]
And that may be the correct way of doing it. I don't know, let's discuss the scenarios.

**Comment by MatteoPiovanelli-Laser** [comment]
As a simple example, I am thinking something like a shorts + tee bundle. I would need to pick the size for both, and it is not necessarily the same attribute for both.
We may have the two attributes set on the bundle itself, I guess, but that does not feel right to me. Also, two items in the bundle may require the same attribute with two different values. 

I feel this may be more an issue of UI/UX than anything. It may be enough to change how products in the bundle are displayed, and then run them through validation of their attributes (maybe this is done already, I don't remember, and I can't verify right now).

**Comment by bleroy** [comment]
Yes, that sounds like a valid scenario. I'm wondering however if it always makes sense to surface all product attributes to the bundle. In your shorts + tee bundle example, what if you also have colors for each, but you want to have only one color choice for the bundle, which selects the same color tee and shorts? In that case, you want to suppress the product attributes and activate a single bundle attribute.

Now you could go crazy and have settings per product attribute in a bundle to surface them or not. This may be overkill, however, and maybe it's good enough to have a simple flag for all of them. We could go either way depending on how important we decide various scenarios are.
